spec.bs

```diff
@@ -26,6 +26,7 @@ spec: html; urlPrefix: https://html.spec.whatwg.org/multipage/
         text: target browsing context; url: webappapis.html#concept-environment-target-browsing-context
         text: active document; url: webappapis.html#active-document
         text: module script; url: webappapis.html#module-script
+        text: global object; url: webappapis.html#concept-realm-global
```

Comment:
This term is already exported so I think you should add:

```
spec:html; type:dfn; for:realm; text:global object
```

in the `<pre class=link-defaults>` above